java/squeek/appleskin/helpers/FoodHelper.java

```diff
@@ -16,6 +16,9 @@
 {
 	public static boolean isFood(ItemStack itemStack)
 	{
+		if (itemStack == null)
+			return false;
+
 		return itemStack.getItem().isFood();
```

Comment:
```suggestion
		return itemStack.getItem() != null and itemStack.getItem().isFood();
```

Reply:
We could check both cases just to be sure...

Reply:
As I understand it, `ItemStack` is not able to be `null` (`ItemStack.EMPTY` is used for that).

Reply:
Makes sense. I changed it.